templates/task/task_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete task?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">
+  </form>
+{% endblock %}

```

Comment:
Add a cancel button, in case the user changes their mind. it should redirect to the previous page the user was on